src/client.js

```diff
@@ -63,10 +63,11 @@ function refresh_token (_refresh_token) {
 }
 
 function revoke_token (token, client_id, type = '') {
-    const scopes_string = cache.find_key('client', {token});
+    const client_key = cache.find_key('client', {token});
+    const server_key = cache.find_key('server', {token});
```

Comment:
Since we don't need these keys until checking if key exists, why not move it to L82? Closer to the usage and avoid unnecessary allocation?

Reply:
I already updated the suggested changes. I also changed the condition on checking keys. It was using 'key' before even though I already renamed the keys. :smile: